src/components/LocationDetails.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropType from "prop-types";
+
+const locationDetails = (props) => {
+  const { city, country } = props;
```

Comment:
Could these props be destructured as they are in the other components? Good to keep things consistent